draft-ietf-rats-eat.md

```diff
@@ -834,7 +834,7 @@ This level indicates that all debug facilities for the entity are permanently di
 The "location" claim gives the geographic position of the entity from which the attestation originates.
 Latitude, longitude, altitude, accuracy, altitude-accuracy, heading and speed MUST be as defined in the W3C Geolocation API {{W3C.GeoLoc}}
 (which, in turn, is based on {{WGS84}}).
-If the entity is stationary, the heading is NaN (floating-point not-a-number).
+If the entity is stationary, the heading is NULL.
```

Comment:
There seems to be no reason for the capitalisation.
(We could use the backticks to show it’s from the CDDL prelude.)

```suggestion
If the entity is stationary, the heading is `null`.
```